docs/src/main/asciidoc/native-reference.adoc

```diff
@@ -206,7 +270,16 @@ These reports are some of the most useful resources when encountering issues wit
 This is useful for getting an approximation on why a method/class is included in the binary.
 However, the text format makes it very difficult to read and can take up a lot of space.
 
-Since Mandrel 21.3.0.0-Final, the call tree is also reported as a group of CSV files.
+Since Mandrel 21.3.0.0, the call tree is also reported as a group of CSV files.
+The CSV output can be enabled by adding `-H:PrintAnalysisCallTreeType=CSV` to the additional native arguments. E.g.
```

Comment:
```suggestion
Since Mandrel 22.0.0.0, the call tree is also reported as a group of CSV files.
The CSV output can be enabled by adding `-H:PrintAnalysisCallTreeType=CSV` to the additional native arguments. E.g.
```

`-H:PrintAnalysisCallTreeType` was introduced in 22.0.0.0, it's not available in 21.3.0.0. In 21.3.0.0 `-Dquarkus.native.enable-reports` (i.e. `-H:PrintAnalysisCallTree`) will produce both the text-formatted call tree and the CSV files.

We might need to consider sticking to 21.3.0.0 in the guide since it's an LTS

Reply:
The updates in this PR are for `main`, which is already past the LTS (2.7 / 21.3) and it's already using Mandrel 22.0.

Reply:
I might send a PR specific for 2.7 branch (the LTS) adjusting things as you suggest above since that's on 21.3.